src/Test/Perf/infra/monitor_and_start_test.csx

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+Task.Run(() => StartWatching());
+Console.WriteLine("Press any key to continue");
+Console.Read();
+
+void StartWatching()
+{
+    FileSystemWatcher watcher = new FileSystemWatcher();
+    watcher.Path = Args[0];
+    watcher.Created += new FileSystemEventHandler(OnChanged);
+    watcher.IncludeSubdirectories = false;
+    watcher.EnableRaisingEvents = true;
+    while (true)
```

Comment:
If I remember correctly the TFS build agent that used to run on our Perf Machines and Perf-rolling machines are having issues and their failures have been tracked down. I think @KevinH-MS put some effort into it to fix it but I dont think it was successful. 

Also this approach is one where we are constantly pinging the build output share for trigger. I dont know this will be our long term approach. Maybe the build agents are the long term solution. But this will get us monitor the builds and get some results. This could be our V1 approach.